### PR TITLE
Jevp Histogram Sync to reflect production code

### PR DESCRIPTION
**Pull request by jml985**
Please do not review quite yet.   I will be inserting files one at a time to check if they pass the tests

### CODE REVIEW

**Review by akioogawa — APPROVED** [review]

File: OnlTools/Jevp/StJevpBuilders/fttBuilder.h [hunk 1]
```diff
@@ -2,23 +2,208 @@
 #include <stdlib.h>
 
 #include "JevpBuilder.h"
-class daqReader;
+#include "DAQ_READER/daqReader.h"
```
Comment (plexoos) [inline OnlTools/Jevp/StJevpBuilders/fttBuilder.h]: This. You don't need to include the whole shebang if the forward declaration is enough.

**Comment by jml985** [comment]
@plexoos - I see you fixed the typo I made in the last commit.  Great that is fully understood, but here is exactly the point where my understanding of git breaks.   How do I get your change to my pull request to be reflected in my version of the JML-FEB01 branch?    Obviously here it doesn't matter as I could easily make the changes myself, but say you made a hundred changes there must be a way to simply accept them and continue to use them without building in conflicts.   The PR isn't yet merged with main, but at the same time it is now different from the PR I personally have made, and which I have in both my working directories and also my github branch.

**Comment by plexoos** [comment]
Jeff, my fix in this commit [e2202e3](https://github.com/star-bnl/star-sw/pull/292/commits/e2202e326c35a02de7a12267112ef5c85f547d1f) *is* on your [jml985:JML-FEB01](https://github.com/jml985/star-sw/tree/JML-FEB01) branch already. I was able to commit directly to your branch because of [this GitHub feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork). This is the default behavior unless you unset the check box "Allow edits from maintainers" when you create a PR.

This PR and therefore your branch [jml985:JML-FEB01](https://github.com/jml985/star-sw/tree/JML-FEB01) is indeed not yet merged into main. You can continue adding changes to this remote branch without conflict when you pull my change from this remote branch into your local branch. The following should do it:

```
git checkout JML-FEB01
git pull
```
At this point you can continue to grow the branch as you like and push the changes from the local JML-FEB01 branch to the remote one as usual.

**Comment by jml985** [comment]
@plexoos - Thank you for the explanation!   This PR is now consistent with the running Jevp code.

**Review by plexoos — APPROVED** [review]

**Comment by starsdong** [comment]
Thanks, Jeff. After the CI builds are checked out, I think we should be able to merge.

Just to be sure, the old requests 
https://github.com/star-bnl/star-sw/pull/230
https://github.com/star-bnl/star-sw/pull/231
https://github.com/star-bnl/star-sw/pull/277
are now all obsolete. We should be able to close them?

**Comment by jml985** [comment]
@starsdong Yes.

**Pull request merged**